core/backend/src/test/standalone/Catalog.test.ts

```diff
@@ -0,0 +1,718 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { assert } from "chai";
+import * as path from "path";
+import { DbResult, Id64, Id64Set, Id64String, Logger, LogLevel } from "@bentley/bentleyjs-core";
+import { Angle, Point2d, Point3d, Range2d, Range3d, YawPitchRollAngles } from "@bentley/geometry-core";
+import {
+  Code, CodeScopeSpec, DefinitionElementProps, GeometricElement2dProps, IModel, PhysicalElementProps, Placement2d, Placement3d, RepositoryLinkProps,
+  SubCategoryAppearance,
+} from "@bentley/imodeljs-common";
+import {
+  BackendLoggerCategory, BackendRequestContext, DefinitionContainer, DefinitionGroup, DefinitionGroupGroupsDefinitions, DefinitionModel,
+  DocumentListModel, Drawing, DrawingCategory, DrawingGraphic, DrawingModel, ECSqlStatement, Element, ElementOwnsChildElements, EntityClassType,
+  IModelDb, IModelJsFs, IModelTransformer, LinkElement, PhysicalElement, PhysicalElementIsOfType, PhysicalModel, PhysicalObject, PhysicalType,
+  RecipeDefinitionElement, RepositoryLink, SnapshotDb, SpatialCategory, TemplateModelCloner, TemplateRecipe2d, TemplateRecipe3d,
+  TypeDefinitionElement,
+} from "../../imodeljs-backend";
+import { IModelTestUtils } from "../IModelTestUtils";
+import { IModelTransformerUtils } from "../IModelTransformerUtils";
+import { KnownTestLocations } from "../KnownTestLocations";
+
+const createClassViews = false; // can set to true to make it easier to debug catalog structure
+
+/** Structure of a Catalog

```

Comment:
Does this documentation comment unattached to any type surface somewhere in the published docs?